### PR TITLE
Update check_conan_version_in_conanfile.rst

### PR DESCRIPTION
**Pull request by blackliner**

### CODE REVIEW

File: howtos/check_conan_version_in_conanfile.rst [hunk 1]
```diff
@@ -11,8 +11,7 @@ Let's have a look at a basic example of this:
 .. code-block:: python
    :caption: conanfile.py
 
-    from conans import ConanFile, CMake, __version__ as conan_version
-    from conans.model.version import Version
+    from conans.tools import Version
```
Comment (memsharded) [inline howtos/check_conan_version_in_conanfile.rst:14]: I don't think this is correct. The code still uses ``conan_version``, ``CMake``, ``ConanFile``, those imports cannot be removed

Comment (blackliner) [inline howtos/check_conan_version_in_conanfile.rst:14]:
```suggestion
    from conans.tools import ConanFile, CMake, Version
```

File: howtos/check_conan_version_in_conanfile.rst [hunk 2]
```diff
@@ -35,8 +34,7 @@ example:
 
 .. code-block:: python
 
-    from conans import ConanFile, tools, __version__ as conan_version
-    from conans.model.version import Version
+    from conans.tools import Version
```
Comment (blackliner) [inline howtos/check_conan_version_in_conanfile.rst:37]:
```suggestion
    from conans.tools import ConanFile, CMake, Version
```

File: howtos/check_conan_version_in_conanfile.rst (hunk 1 above)
Comment (blackliner) [inline howtos/check_conan_version_in_conanfile.rst:14]:
```suggestion
    from conans import ConanFile, CMake
    from conans.tools import Version
```

File: howtos/check_conan_version_in_conanfile.rst (hunk 2 above)
Comment (blackliner) [inline howtos/check_conan_version_in_conanfile.rst:37]:
```suggestion
    from conans import ConanFile, tools
    from conans.tools import Version
```

File: howtos/check_conan_version_in_conanfile.rst [hunk 3]
```diff
@@ -35,8 +35,8 @@ example:
 
 .. code-block:: python
 
-    from conans import ConanFile, tools, __version__ as conan_version
-    from conans.model.version import Version
+    from conans import ConanFile, tools
```
Comment (blackliner) [inline howtos/check_conan_version_in_conanfile.rst:38]:
```suggestion
    from conans import ConanFile, tools, __version__ as conan_version
```

File: howtos/check_conan_version_in_conanfile.rst [hunk 4]
```diff
@@ -11,8 +11,8 @@ Let's have a look at a basic example of this:
 .. code-block:: python
    :caption: conanfile.py
 
-    from conans import ConanFile, CMake, __version__ as conan_version
-    from conans.model.version import Version
+    from conans import ConanFile, CMake
```
Comment (blackliner) [inline howtos/check_conan_version_in_conanfile.rst:14]:
```suggestion
    from conans import ConanFile, CMake, __version__ as conan_version
```

**Pull request merged**